io/net/tcpdump.py

```diff
@@ -59,17 +82,42 @@ def test(self):
         obj = process.SubProcess(cmd, verbose=False, shell=True)
         obj.start()
         output_file = os.path.join(self.outputdir, 'tcpdump')
-        cmd = "tcpdump -i %s -n -c %s -w '%s'" % (self.iface, self.count,
-                                                  output_file)
+        cmd = "tcpdump -i %s -n -c %s" % (self.iface, self.count)
+        if self.option in ('host', 'src'):
+            cmd = "%s %s %s" % (cmd, self.option, self.host_ip)
+        elif self.option == "dst":
+            cmd = "%s %s %s" % (cmd, self.option, self.peer_ip)
+        elif self.option == "tcp":
+            obj = self.nping('tcp')
+            obj.start()
+            cmd = "%s %s" % (cmd, self.option)
+        elif self.option == "udp":
+            obj = self.nping('udp')
+            obj.start()
+            cmd = "%s %s" % (cmd, self.option)
+        elif self.option == "icmp":
+            obj = self.nping('icmp')
+            obj.start()
+            cmd = "%s %s" % (cmd, self.option)
+        else:
+            cmd = "%s %s" % (cmd, self.option)
+        cmd = "%s -w '%s'" % (cmd, output_file)
         for line in process.run(cmd, shell=True,
                                 ignore_status=True).stderr.decode("utf-8") \
                                                    .splitlines():
-            if "packets dropped by interface" in line:
+            if "packets dropped by kernel" in line:
```

Comment:
what is the reason this is changed ?

Reply:
beacuse after running command not getting, packets dropped by interface.i am getting like this,so i changed.
10 packets captured
10 packets received by filter
0 packets dropped by kernel

Reply:
can you run for more than 500 packets and check please ?

Reply:
Packets dropped by kernel might include other interfaces also. We are interested only in packets dropped by one interface under test.